Resources/Styles/package_list.scss

```diff
@@ -22,9 +22,9 @@
   li {
     background-color: var(--package-list-background);
     border-left: 3px var(--package-list-border) solid;
+    display: grid;
```

Comment:
I don't think this line is necessary. The `a` in the `two_column` class which you have as a child of this is a grid which is what you're using to position the text. This may be left over from an earlier attempt?